src/charm.py

```diff
@@ -150,6 +180,27 @@ def _pebble_layer(self) -> pebble.LayerDict:
             },
         }
 
+    @property
+    def fqdns(self) -> list[str]:
+        """Get the FQDNS for the charm units.
+
+        Returns: the list of FQDNs for the charm units.
+        """
+        unit_name = self.unit.name.replace("/", "-")
+        app_name = self.app.name
+        addresses = [f"{unit_name}.{app_name}-endpoints"]
+        peer_relation = self.model.relations[WAZUH_PEER_RELATION_NAME]
+        if peer_relation:
+            relation = peer_relation[0]
+            # relation.units will contain the units after the relation-joined event.
+            # since a relation-changed is emitted for every relation-joined event,
+            for u in relation.units:
+                # <unit-name>.<app-name>-endpoints.<model-name>.svc.cluster.local
```

Comment:
I'm not following this comment?

Reply:
Perhaps it wasn't committed? It still seems to be the same

Reply:
How about now?

Reply:
I see the comment above getting updated, I was referring to `# <unit-name>.<app-name>-endpoints.<model-name>.svc.cluster.local`

Reply:
Ah, perhaps add a note explaining that is what it is?

Reply:
Done